ifupdown2/addons/vxlan.py

```diff
@@ -150,14 +156,18 @@ def reset(self):
 
     def syntax_check(self, ifaceobj, ifaceobj_getfunc):
         if self._is_vxlan_device(ifaceobj):
-            if not ifaceobj.get_attr_value_first('vxlan-local-tunnelip') and not self._vxlan_local_tunnelip:
-                self.logger.warning('%s: missing vxlan-local-tunnelip' % ifaceobj.name)
+            if not ifaceobj.get_attr_value_first('vxlan-local-tunnelip') and not self._vxlan_local_tunnelip \
+                    and not ifaceobj.get_attr_value_first('vxlan-local-tunnelip6') and not self._vxlan_local_tunnelip6:
+                self.logger.warning('%s: missing vxlan-local-tunnelip or vxlan-local-tunnelip6' % ifaceobj.name)
                 return False
             return self.syntax_check_localip_anycastip_equal(
                 ifaceobj.name,
                 ifaceobj.get_attr_value_first('vxlan-local-tunnelip') or self._vxlan_local_tunnelip,
                 self._clagd_vxlan_anycast_ip
-            )
+            ) or self.syntax_check_localip_anycastip_equal(
+                ifaceobj.name,
+                ifaceobj.get_attr_value_first('vxlan-local-tunnelip6') or self._vxlan_local_tunnelip6,
+                self._clagd_vxlan_anycast_ip)
         return True
 
     def syntax_check_localip_anycastip_equal(self, ifname, local_ip, anycast_ip):
```

Comment:
The syntax check will still report "tunnelip" even if tunnelip6 is identical to clagd-vxlan-anycast-ip.

I am unsure wether clagd-vxlan-anycast-ip could even be IPv6 or (if that is not the case) how this should be properly refactored.